examples/semver.peggy

```diff
@@ -57,4 +55,4 @@ digit
   = [0-9]
 
 positiveDigit
-  = [1-9]
\ No newline at end of file
+  = [1-9]
```

Comment:
Any other places where `[1-9]` is used?

Reply:
I'd like to stay as close as possible to the original grammar.